Libplanet/Net/Protocols/KademliaProtocol.cs

```diff
@@ -18,48 +18,38 @@ internal class KademliaProtocol : IProtocol
         private readonly Address _address;
 
         private readonly Random _random;
-        private readonly RoutingTable _routing;
-        private readonly int _tableSize;
-        private readonly int _bucketSize;
+        private readonly RoutingTable _table;
         private readonly int _findConcurrency;
 
         private readonly ILogger _logger;
 
         public KademliaProtocol(
+            RoutingTable table,
             ITransport transport,
             Address address,
-            ILogger logger,
-            int? tableSize,
-            int? bucketSize,
             int findConcurrency = Kademlia.FindConcurrency,
             TimeSpan? requestTimeout = null)
         {
             _transport = transport;
-            _logger = logger;
+            _logger = Log.ForContext<KademliaProtocol>();
 
             _address = address;
             _random = new System.Random();
-            _tableSize = tableSize ?? Kademlia.TableSize;
-            _bucketSize = bucketSize ?? Kademlia.BucketSize;
             _findConcurrency = findConcurrency;
-            _routing = new RoutingTable(_address, _tableSize, _bucketSize, _random, _logger);
+            _table = table;
             _requestTimeout =
                 requestTimeout ??
-                TimeSpan.FromMilliseconds(Kademlia.IdleRequestTimeout);
+                TimeSpan.FromMilliseconds(5000);
```

Comment:
Is it okay we use `5000` directly instead of named constant?

Reply:
I thought it is weird that timeout value is located at Kademlia constants.